src/Illuminate/Database/Connection.php

```diff
@@ -1347,6 +1347,24 @@ public function getDriverName()
         return $this->getConfig('driver');
     }
 
+    /**
+     * Get a human-readable name for the given connection.
+     */
+    public function getConnectionName(): string
+    {
+        return $this->getName().'('.$this->getDriverName().')';
+    }
+
+    /**
+     * Get the number of open connections for a database.
+     *
+     * @return int|null
+     */
+    public function getConnectionCount(): ?int
```

Comment:
Reading the code further, I realize that this method should be in `Database\Query\Builder` and use the `Grammar` of each DBMS. I'll make this change.